### ISSUE TITLE
Mobs falling apart when hit

### ISSUE THREAD

**Issue by chloexburman** [SupremeObsidian/ProjectManager#8]
All of our mobs' geometry comes apart when they are hit by a player. This might be a problem in the future, especially if it's something that has to do with how we animate or create our models.

**Comment by UUBlueFire** [comment]
According to @violetskyzz It is impossible to fix this. Is this correct?

**Comment by dryter97** [comment]
Kinda based on the model animation himself should read about those kind of
animation. I saw that before and so far everyone was able to make a
workaround

**Comment by UUBlueFire** [comment]
Private conversation with @violetskyzz :
> violetskyzz
There is a work around to a certain extent but I have talked to the creators of mythic mobs and they said it is normal for the bones to come apart when the model is turning a corner
Slightly come apart, obviously when the head is flying off ten feet that is fixable. And I have fixed them they just haven’t been implemented.

> BlueFire
Okay awesome

> violetskyzz
But I mean if @dryter97 thinks he can make them stop doing that he can go for it. I haven’t been able to find an answer from anyone

> BlueFire
That's all good. I'm just putting it all in the ProjectManager because i would like everyone to be aware of it

> violetskyzz
I’m wondering if there’s an animation state I’m missing for specifically when they are hit
Cuz I have currently : idle. Walk, attack, fall, death. But maybe there is one I’m not aware of
